dash/test/ThreadsafetyTest.cc

```diff
@@ -0,0 +1,131 @@
+
```

Comment:
Correct, mainly because they are only used internally and not exposed to the outside world. It seems I cannot include `dash/dart/base/atomic.h` in a test file.